Bottom: a00a46543c9c1a37bb586b19c01a0d8ab1d80891
Top:    de7eefc2d104178a3f161fc03631a12e8adc1c85
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-08 09:36:02 +0800

Refresh of autofs-5.0.7-fix-potential-null-dereference-in-lookup_mount.patch

---

diff --git a/modules/lookup_yp.c b/modules/lookup_yp.c
index d70605a..e99e3c0 100644
--- a/modules/lookup_yp.c
+++ b/modules/lookup_yp.c
@@ -701,6 +701,7 @@ int lookup_mount(struct autofs_point *ap, const char *name, int name_len, void *
 			cache_writelock(mc);
 			cache_update_negative(mc, source, key, ap->negative_timeout);
 			cache_unlock(mc);
+			return NSS_STATUS_TRYAGAIN;
 		}
 	 }
